microsoft-edge/devtools-guide-chromium/console/api.md

```diff
@@ -20,12 +20,27 @@ ms.date: 05/04/2021
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
+# Console object API Reference
+<!-- orig:
 # Console API reference
+-->
 
-The **Console** tool is helpful when you complete multiple tasks in the DevTools.  APIs are available to include in your scripts. Convenience methods are only available for use in the **Console** tool, such as the `debug()` and `monitorEvents()` methods.  For more information on getting started with the **Console**, see [Get started with logging messages to the Console](console-log.md).  For more information on the convenience methods in the **Console**, see [Console Utilities API Reference](utilities.md).
+Use the `console` object's methods to write messages to the **Console** from your JavaScript.
 
----
+You can also enter these methods into the **Console**.  For example, in the **Console**, to enter a `console` method that takes a variable:
+
+1. In the **Sources** tool, in the Debugger, set a breakpoint in your JavaScript code.
+
+1. In the Debugger, step through your code.
 
+1. When you are in a valid context so that the desired variable is in-scope, enter a method of the `console` object into the **Console** tool.  The result is displayed in the **Console**.
+
+* See [Get started with logging messages to the Console](console-log.md) for an interactive introduction.
```

Comment:
Reviewer, 
These links are from upstream version of article.  This PR copied them to a new See also section below, too.  Remove them from here?  The recently improved TOC with true titles, plus improved titles in this PR & previous PR, along with improved intro in this article, means that these links are not needed anymore; the titles are now self-differentiating, and the TOC bucket is clearer.  If these links are here, at top of article, that suggests having a See also at top of article, but there's already that at bottom of article, now.